test/data/serializers/user_serializer_test.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:memo/data/serializers/memo_difficulty_parser.dart';
+import 'package:memo/data/serializers/user_serializer.dart';
+import 'package:memo/domain/enums/memo_difficulty.dart';
+import 'package:memo/domain/models/user.dart';
+
+void main() {
```

Comment:
In my eyes, what you are suggesting is already being tested. Without optional properties means only required properties, which are also tested with the optional properties test case.

Reply:
I was reading the `User` and `UserSerializer` and I realized that both properties `executionsAmounts` and `timeSpentInMillis` cannot be null, so my suggestion doesn't apply to this use-case.

But what I was trying to suggest is that, IF both properties could be null, IMO we would have to test both extreme conditions: with and without all optional properties, because we may find wrong force-unwrapping situations in both cases. In this test as an example, if the properties were optionals, the possibility of `UserKeys.executionsAmounts` doesn't exists in `rawUser` would be not tested in this scenario